travis/large-requirements.txt

```diff
@@ -28,7 +28,7 @@ typing==3.6.6
 pysftp==0.2.9
 keras==2.3.1
 attrdict==2.0.0
-azureml-sdk==1.0.17; python_version >= "3.0"
+azureml-sdk==1.2.0; python_version >= "3.0"
```

Comment:
Addressed, though I dropped the python version qualifier as MLflow dropped py2 as well, and i slimmed the dep to azureml-core instead of azureml-sdk which is a metapackage for more things than we need here